pkg/logger/logger.go

```diff
@@ -23,87 +23,53 @@ func (logger syncLogger) Sync() {
 	logger.SyncImplementation()
 }
 
-func ConfigureDevelopmentLogger(level string) (Logger, error) {
+func ConfigureDevelopmentLogger(level string, syncs ...io.Writer) (Logger, error) {
```

Comment:
@tim-thacker-nullify why do we have two loggers, one for prod and one for dev again?

Reply:
dev pretty prints the logs which are human readable and prod does structured json logging which is more machine readable